diagnosis/getkuberneteslogs.sh

```diff
@@ -40,8 +150,13 @@ function printUsage
     echo "  -i, --identity-file             RSA private key tied to the public key used to create the Kubernetes cluster (usually named 'id_rsa')"
     echo "  -m, --master-host               A master node's public IP or FQDN (host name starts with 'k8s-master-')"
     echo "  -d, --vmd-host                  The DVM's public IP or FQDN (host name starts with 'vmd-')"
+    echo "  --spn-client-id                 Service Principle client Id used to create the Kubernetes cluster"
+    echo "  --spn-client-secret             Service Principle client secret used to create the Kubernetes cluster"
+    echo "  -t, --tenant-id                 Tenant id"
+    echo "  -l, --location                  Location"
     echo "  -n, --user-namespace            Collect logs for containers in the passed namespace (kube-system logs are always collected)"
     echo "  --all-namespaces                Collect logs for all containers. Overrides the user-namespace flag"
+    echo "  --upload-storage-account        Upload logs to storage account. Specify Y to upload logs to Storage Account. Specify N to disable upload logs to Storage Account "
```

Comment:
I think all this `Specify Y to upload logs to Storage Account. Specify N to disable upload logs to Storage Account` can be replaced by `[Y,N]`

Reply:
I may be easier to assume `yes` if the flag is present. And assume `no` if not present

Otherwise we need to make sure the script can handle all these: y, yes, Y, YES, Yes